README.md

```diff
@@ -12,13 +12,9 @@
 If you are not acquainted with Rails, it may help to go through the [Getting Started with Rails](http://guides.rubyonrails.org/v5.1/getting_started.html) tutorial for Rails 5.1.
 If you wish to learn Ruby, try out the [quickstart](https://www.ruby-lang.org/en/documentation/quickstart/) guide provided on their website, or if you are already acquainted with various programming languages, try the [Learn X in Y minutes](https://learnxinyminutes.com/docs/ruby/) tutorial for Ruby.
 
-This README mostly focuses on how to get started developing this project with Ubuntu – commands including `apt` or `apt-get` use the Debian/Ubuntu package manager.
-If you are using another flavor of UNIX (another Linux, such as Arch Linux, or macOS), you may find that a lot of the commands are similar but specifics will likely vary – the links to the individual dependency manuals may help you in this case.
-
-If you're using Windows, you may find your best bet – for a variety of reasons – is to set up a virtual machine running Ubuntu where you can do your development.
-This is not completely trivial, but if you choose to do so it would be worth looking into [Oracle VirtualBox](https://www.virtualbox.org/).
-
-It should be possible to develop this project otherwise on a Windows machine, but the steps to install all the dependencies on Windows are quite different, and a lot of the steps will differ from those listed here.
+This README mostly focuses on how to get started developing this project with
+Docker -- in any environment with Docker installed you should be able to have
+the Glowfic server up and running.
```

Comment:
Nitpick for here and other places: I prefer linebreaks to occur at the end of sentences in documentation, so rephrasing sentences won't cause diffs for the whole paragraph (only the lines modified). Could you modify the hard wrapping accordingly?

Reply:
Done =)